doc/user/content/sql/create-sink.md

```diff
@@ -139,6 +140,15 @@ You can find the topic name and other metadata for each Kafka sink by querying [
 
 To achieve its exactly-once processing guarantees, Materialize needs to store some internal metadata in an additional *progress topic*. This topic is shared among all sinks that use a particular Kafka connection. The name of this progress topic can be specified when [creating a connection](/sql/create-connection); otherwise, a default is chosen based on the Materialize environment `id` and the connection `id`. In either case, Materialize will attempt to create the topic if it does not exist. The contents of this topic are not user-specified.
 
+## Best practices
+
+### Sizing a sink
+
+Some sinks are low traffic and require relatively few resources, while others
+are high traffic and require hefty resource allocations. You choose the amount
+of CPU and memory available to a sink using the `SIZE` option. You cannot yet
+change the size of a sink after creation.
```

Comment:
Sounds like this is about to change: https://github.com/MaterializeInc/materialize/pull/15311